xapian-bindings/android/android.i

```diff
@@ -0,0 +1,401 @@
+%module(directors="1") xapian
```

Comment:
I believe I've managed to build android bindings from the standard java bindings subdirectory by adding a custom rule to build `xapianlib.aar` and passing suitable options when building the bindings.  However, I'm now stuck at compiling the example app so I can actually test the `xapianlib.aar` I built.